src/Console/Command/ReadmeCommand.php

```diff
@@ -213,7 +213,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $help = $command->getHelp();
         $help = str_replace('%command.full_name%', 'php-cs-fixer.phar '.$command->getName(), $help);
         $help = str_replace('%command.name%', $command->getName(), $help);
+        $help = preg_replace('#<<(</.*?>)#', '\\\\$1', $help);
         $help = preg_replace('#</?(comment|info)>#', '``', $help);
+        $help = preg_replace('#`(``.+?``)`#', '$1', $help);
```

Comment:
The triple backticks are introduced by the line just before this one, but I can probably merge both regular expressions.